src/NumericalAlgorithms/Interpolation/Callbacks/ObserveSurfaceIntegrals.hpp

```diff
@@ -69,32 +123,36 @@ struct ObserveSurfaceIntegrals {
         get<StrahlkorperTags::NormalOneForm<Frame>>(box),
         get<StrahlkorperTags::Radius<Frame>>(box),
         get<StrahlkorperTags::Rhat<Frame>>(box));
-    std::vector<double> time_and_integrals(tmpl::size<TagsToObserve>::value +
-                                           1);
+
+    // The +1 in the sizes below is because `TagsToObserve` contains
+    // only the integrals, but both `legend` and `time_and_integrals` contain
+    // time in addition to the integrals.
     std::vector<std::string> legend(tmpl::size<TagsToObserve>::value + 1);
+    std::array<double, tmpl::size<TagsToObserve>::value + 1>
+        time_and_integrals{};
     time_and_integrals[0] = temporal_id.time().value();
```

Comment:
So I understand, why is this not `gal::at`? Is this safe because you ask for the 0 element?

Reply:
yes, that's exactly right :)